common/meta/raft-store/src/config.rs

```diff
@@ -144,8 +147,17 @@ impl RaftConfig {
         <Self as Parser>::parse_from(&Vec::<&'static str>::new())
     }
 
-    pub fn raft_api_addr(&self) -> String {
-        format!("{}:{}", self.raft_api_host, self.raft_api_port)
+    /// Support ip address and hostname
+    pub async fn raft_api_addr(&self) -> Result<String> {
+        match IPAddress::parse(self.raft_api_host.as_str()) {
```

Comment:
Seems the parsed `_ip_address` is useless, I think the [is_valid](https://docs.rs/ipaddress/latest/ipaddress/struct.IPAddress.html#method.is_valid) function is enough here.

Reply:
ok,Let me do it.